app/pubspec.yaml

```diff
@@ -33,13 +33,13 @@ dependencies:
   yaml: '^2.1.12'
   # pana version to be pinned
   # keep in-sync with app/lib/shared/versions.dart
-  pana: '0.11.1'
+  pana: '0.11.2'
```

Comment:
Oops! Uh...I'll let you run w/ this @isoos

Reply:
You'll need to change the `shared/versions.dart` too, and with that, update the runtimeVersion and the related test. Feel free to leave it out, I'll upgrade it (but wanted to do it with another change included, but it is not that important to bundle it).